bleachbit.py

```diff
@@ -25,12 +25,38 @@
 import os
 import sys
 
+import bleachbit.Unix
```

Comment:
@rados 
This import causes an error on Windows

https://ci.appveyor.com/project/az0/bleachbit/builds/45868580#L698

Reply:
Fixed.